### DISCUSSION TITLE
Zoom: “Fit to window”

### DISCUSSION THREAD

**Discussion by ssh4net** [houmain/gpupad#5] [Q&A] [answered]
Did GPUpad have any way to fit viewport to window it open? Or at least set desired scale.

When working with bigger textures and viewports that match texture sometime hard to maximize render into the window.

Thank you!

**Comment by houmain** [comment, marked as answer]
Hi Vlad, 
I added a zoom control to the toolbar and the textures are now automatically zoomed to fill the window. 
One has to hold the Control key now for zooming with the mouse wheel.
It's a cool feature. Thanks for the suggestion!

**Reply by ssh4net** [reply]
Amazing! Thank you!

Maybe worth add `(Use Ctrl to zoom with mouse wheel)` after numeric input?
It can be tricky without good help system guess such modifier :)
